### PR TITLE
Bump up webpack to v5

### PR DESCRIPTION
**Pull request by mtgto**
https://webpack.js.org/migrate/5/

It seems to be faster than v4 (and old loaders and plugins).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,9 +70,8 @@
     "@types/webdriverio": "4.13.3",
     "@typescript-eslint/eslint-plugin": "3.4.0",
     "@typescript-eslint/parser": "3.4.0",
-    "clean-webpack-plugin": "0.1.19",
```
Comment (mtgto) [inline package.json:73]: webpack v5 has an option [output.clean](https://webpack.js.org/configuration/output/#outputclean).

File: webpack.config.js [hunk 2]
```diff
@@ -53,17 +53,24 @@ module.exports = (env, argv) => {
...
     },
     commonConfig
   );
 
   const rendererConfig = Object.assign(
     {
-      devtool: argv.mode === "production" ? "inline-source-map" : "eval-source-map",
+      // mini-css-extract-plugin does not support "cheap-source-map".
+      // See https://github.com/webpack-contrib/mini-css-extract-plugin/issues/529
+      devtool: argv.mode === "production" ? "inline-source-map" : "source-map",
```
Comment (mtgto) [inline webpack.config.js:65]: cssのsourcemap別になくてもいいかも、と思わなくもないです。

File: webpack.config.js [hunk 3]
```diff
@@ -53,17 +53,24 @@ module.exports = (env, argv) => {
...
       target: "electron-renderer",
       entry: "./src/renderer/app.tsx",
       output: {
+        clean: {
+          keep(asset) {
+            return asset === "main.js";
```
Comment (mtgto) [inline webpack.config.js:71]: Delete all output files except "main.js".

File: webpack.config.js [hunk 4]
```diff
@@ -77,21 +84,23 @@ module.exports = (env, argv) => {
...
+                options: {
+                  sourceMap: true
+                }
+              }
             ]
           },
           {
             test: /\.(png|ttf|eot|svg|woff|woff2)(\?.+)?$/,
-            loader: "url-loader"
+            type: "asset/inline"
```
Comment (mtgto) [inline webpack.config.js:98]: url-loader is deprecated since webpack v5.

**Review by hokaccha — APPROVED** [review]

**Pull request merged**